server/api/users.js

```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+const router = express.Router();
+const User = require("../database/models/User.js");
```

Comment:
Should I rename this to "Users" with an "s"?

Reply:
Yeah sure, if you still can lol

Reply:
```js
const Users = require("../database/models/User.js");
```

Reply:
Yeah looks good

Reply:
alr